<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>427</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>427</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>404</width>
    <height>427</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyPass Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/password.png</normaloff>icons/password.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(24,24,36);

}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-40</y>
     <width>441</width>
     <height>471</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>471</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>441</width>
     <height>471</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(24,24,36);
border-color: rgb(16, 20, 20);
</string>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>141</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/users.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_user_line">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter username to login</string>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-radius:7px;
	border:1px solid #18ab29;
color: gray;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string> Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_password_line">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter password :)</string>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-radius:7px;
	border:1px solid #18ab29;
color: gray;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string> Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>340</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>12</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Login to PyPass</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: white;
font: 87 12pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="status">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>400</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string>    </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="iconsref.qrc"/>
 </resources>
 <connections/>
</ui>
